toolkit/tools/scheduler/schedulerutils/implicitprovides.go

```diff
@@ -61,16 +61,16 @@ func replaceNodesWithProvides(res *BuildResult, pkgGraph *pkggraph.PkgGraph, pro
 	// - By making a run node the parent node, it will inherit the identical runtime dependencies of the already setup node.
 	for _, node := range pkgGraph.AllRunNodes() {
 		// We need to filter out any non-local run nodes since cached remote nodes are not acceptable parent nodes for collapsed nodes.
-		if node.Type == pkggraph.TypeLocalRun && rpmFileProviding == node.RpmPath {
-			logger.Log.Debugf("Linked implicit provide (%s) to run node (%s)", provides, node.FriendlyName())
+		if (node.Type == pkggraph.TypeLocalRun || node.Type == pkggraph.TypePreBuilt) && rpmFileProviding == node.RpmPath {
```

Comment:
@PawelWMS thoughts on keeping this? I can't think of a reason we would end up with implicit nodes that link back to toolchain nodes now that the prebuilt stuff is fixed.

Reply:
I also can't think of a reason for that but I'm not too familiar with the logic in this function. I think we can leave the original version for now but my reason would be to not introduce more changes until we're sure the builds have stabilized a bit.